README.md

```diff
@@ -71,10 +71,11 @@ In addition to the resources mentioned above, the following Kubernetes resources
 The ECK related custom resources are included in those namespaces as well: 
 * Agent
 * ApmServer
+* Beat
 * ElasticMapsServer
 * Elasticsearch
 * EnterpriseSearch
 * Kibana
 
 ### Logs
-In the operator namespaces (`-o, --operator-namespaces`) all logs are collected, while in the workload resource namespaces only those logs that originate in Pods managed by ECK are collected .
\ No newline at end of file
+In the operator namespaces (`-o, --operator-namespaces`) all logs are collected, while in the workload resources namespaces only logs from Pods managed by ECK are collected.
```

Comment:
```suggestion
In the operator namespaces (`-o, --operator-namespaces`) all logs are collected, while in the workload resource namespaces only logs from Pods managed by ECK are collected.
```

I don't think we need the plural even though there are ore than one per namespace.